dev/io.openliberty.microprofile.telemetry.internal.common/src/io/openliberty/microprofile/telemetry/internal/common/info/OpenTelemetryInfoFactoryImpl.java

```diff
@@ -185,16 +214,78 @@ private static boolean checkDisabled(Map<String, String> oTelConfigs) {
     private HashMap<String, String> getTelemetryProperties() {
         try {
             Config config = ConfigProvider.getConfig();
+
             HashMap<String, String> telemetryProperties = new HashMap<>();
             for (String propertyName : config.getPropertyNames()) {
                 if (propertyName.startsWith("otel") || propertyName.startsWith("OTEL")) {
                     String normalizedName = propertyName.toLowerCase().replace('_', '.');
+
                     config.getOptionalValue(normalizedName, String.class)
-                        .ifPresent(value -> telemetryProperties.put(normalizedName, value));
+                          .ifPresent(value -> telemetryProperties.put(normalizedName, value));
+
                 }
             }
 
-            telemetryProperties.putAll(openTelemetryVersionedConfiguration.getTelemetryPropertyDefaults());
+            return telemetryProperties;
+        } catch (Exception e) {
+            Tr.error(tc, Tr.formatMessage(tc, "CWMOT5002.telemetry.error", e));
+            return new HashMap<String, String>();
+        }
+    }
+
+    private HashMap<String, String> getServerTelemetryProperties() {
```

Comment:
There's a bug here. Its possible to have two config properties with the same identity but different String. e.g. my.property and MY_PROPERTY will be normaized to the same normalizedName.

Either mpConfig needs to handle that for us, or we need to redo this code to get the properties so the highest priority goes in first and use putIfAbsent. When I tried that we got an OSGi error so that needs doing later.

Since this is a pre-existing bug we can handle it in a follow up